build.gradle

```diff
@@ -15,15 +15,16 @@ buildscript {
         log4j_version = constants.getProperty("log4jVersion")
         slf4j_version = constants.getProperty("slf4jVersion")
         corda_platform_version = constants.getProperty("platformVersion")
+        publicArtifactURL='https://download.corda.net/maven'
```

Comment:
Just for consistency with other properties, should this value be added to the [gradle.properties](https://github.com/corda/bn-extension/blob/release/1.2/gradle.properties)?